dbServer/Dockerfile

```diff
@@ -7,6 +7,9 @@ RUN \
     # Check for mandatory build arguments
         : "${GRAPHDB_ZIP:?Provide the downloaded GraphDB Free ZIP file}"
 
+ENV GRAPHDB_WORKBENCH_EXTERNAL_URL=http://localhost/modelujeme/sluzby/dbServer
```

Comment:
According to the docs, the workbench external url param is deprecated, and was replace by the other one (external-url).